layouts/default.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div>
     <NavigationMain />
+    <NavigationLogin />
```

Comment:
Does this look nice here? plan was to put it into the footer, but if it looks fine in the header it's also ok

Reply:
we can keep it here only for now, since Matic is currently working on a better template for the demo. when that is done we can merge and also move the location of login/logout button maybe on the footer section or wherever it looks best.

Reply:
@harrypango no, we want to keep this template and have 2 demo templates in the future. please put it into the footer then.